vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -235,12 +235,23 @@ void LedgerImpl::OnPostData(
     return;
   }
 
-  std::vector<std::map<std::string, std::string>> twitchParts;
-  if (TWITCH_MEDIA_TYPE == type) {
+  if (type == TWITCH_MEDIA_TYPE) {
+    std::vector<std::map<std::string, std::string>> twitchParts;
```

Comment:
Good place to use `auto` here, if possible.

Reply:
my only concerns with `auto` is that I have no idea what type `twitchParts` is 😃

Reply:
Haha, well just consider it a nit not a blocker ;-)